pkg/server.go

```diff
@@ -57,19 +56,32 @@ func Start(cfg *Config) {
 		timeout = pluginConfig.Timeout
 	}
 
+	logrusLevel, err := logrus.ParseLevel(cfg.LogLevel)
+	if err != nil {
+		logrus.WithError(err).Fatal("unable to set the log level")
+		logrusLevel = logrus.ErrorLevel
+	}
+
 	httpServer := &http.Server{
-		Handler:      loggedRouter,
+		Handler:      router,
 		Addr:         fmt.Sprintf(":%d", cfg.Port),
 		TLSConfig:    tlsConfig,
 		ReadTimeout:  timeout,
 		WriteTimeout: timeout,
 	}
 
+	if logrusLevel == logrus.TraceLevel {
```

Comment:
isn't enough to set the log level? I think route logs will be on the info level so by default they'll be skipped

Reply:
The handlers.LoggingHandler comes from the gorilla/handlers package, which doesn't respect or use the logrus internally. Setting the log level for logrus has no impact on the route logs

Reply:
Ok got it. Thnx